dll/win32/syssetup/lang/pt-PT.rc

```diff
@@ -202,7 +202,7 @@ BEGIN
      *   samsrv.dll:IDS_USER_ADMINISTRATOR_NAME synchronized.
      *   Also check the IDD_COMPUTERPAGE dialog.
      */
-    IDS_ADMINISTRATOR_NAME "Administrator"
+    IDS_ADMINISTRATOR_NAME "Administrador"
```

Comment:
What's the comment just one line above about?

Reply:
OPPS!